plugin/editorconfig.vim

```diff
@@ -234,6 +235,7 @@ function! s:UseConfigFiles() abort " Apply config to the current buffer {{{1
         endif
     endfor
 
+    let b:editorconfig_had_config = 1
```

Comment:
- This is to indicate whether EditorConfig actually ran against the buffer, right?  I don't think this should be set if we hit the error case below.
- How about calling it `b:editorconfig_applied` instead?  Just because the core was run doesn't mean that the file actually matched any EditorConfig rule.

Reply:
Sounds good. Renamed and avoided setting in error cases.